lib/util/enum_util.ex

```diff
@@ -24,4 +27,24 @@ defmodule Antikythera.EnumUtil do
   defun map_with_context(e :: Enum.t, c :: context, fun :: (item, context -> {item, context})) :: [item] do
     Enum.map_reduce(e, c, fun) |> elem(0)
   end
+
+  @doc """
+  `Enum.find/3` extension for Antikythera.
+
+  If an element is not found, it will crash instead of returning `nil`.
+  Also, only the default value is allowed for the second argument.
```

Comment:
Comment on the default value (l.35) is not needed, as the behaviour of this function where no matching element is found is already explained in l.34.
The following 1 line would look OK for me:
```
A variant of `Enum.find/2` that raises an exception when no matching element is found.
```

Reply:
fixed in 3f9f446